Package.swift

```diff
@@ -0,0 +1,15 @@
+// swift-tools-version:4.0
+import PackageDescription
+
+let package = Package(
+    name: "WallE",
```

Comment:
It can't be `Wall-E`?

Reply:
I avoided on purpose the addition of `-`.

Reply:
LGTM